homeassistant/components/netgear/__init__.py

```diff
@@ -18,6 +22,21 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except CannotLoginException as ex:
         raise ConfigEntryNotReady from ex
 
+    port = entry.data.get(CONF_PORT)
+    ssl = entry.data.get(CONF_SSL)
+    if port != router.port or ssl != router.ssl:
+        data = entry.data.copy()
+        data[CONF_PORT] = router.port
+        data[CONF_SSL] = router.ssl
+        hass.config_entries.async_update_entry(entry, data=data)
+        _LOGGER.info(
+            "Netgear port-SSL combination updated from (%i, %r) to (%i, %r), this should only occur after a firmware update",
```

Comment:
Please break long strings around 88 characters per line.

Reply:
corrected in PR https://github.com/home-assistant/core/pull/64428